### DISCUSSION TITLE
New eOS  variants

### DISCUSSION THREAD

**Discussion by wasantosfi** [elementary/triage#490] [Ideas]
A good option to eOS 8 would have 2 more variants together the current one.

**Mini**:
-- Less bloatwares (user choose what he/she want during the system installation process)
-- Lightweight kernel and drivers

**Privacy**:
-- Change default web browser: Gnome Browser -> Brave
-- Change default e-mail client: Mail -> Thunderbird
-- Add comm encryption: GnuPG
-- Add disk encryption: VeraCrypt
-- Add VPN client: OpenVPN and WireGuard
-- Add password manager: Bitwarden or KeePassXC
-- Add permanently deletion: File Shredder
-- Add end-to-end encrypted file-sharing: OnionShare
-- Add 2FA tool: Authenticator
-- Add snapshot tool: Timeshift
-- Add anon connections: Tor

**Comment by danirabbit** [comment]
Hey thanks for your suggestions. In the future I recommend opening separate discussion posts and individual feature requests where possible. It's very hard to track discussions that cover a lot of different topics and they often duplicate and overlap other discussions, feature requests, etc.

Can you be more specific about what you mean by "bloatware"? What things do you think we're shipping right now that are unnecessarily and should be removed?

Can you elaborate on what a "lightweight" kernel is or provide examples of distributions shipping what you would consider to be a "lightweight" kernel? Could you elaborate on what you mean regarding drivers?

I definitely would never recommend anyone use Brave browser. It has a super sketchy history rooted in homophobia and cryptocurrency scams. GNOME Web is definitely the more privacy-respecting choice in my opinion.

Can you elaborate on why you think Thunderbird is more privacy respecting than elementary Mail?

There's an open feature request for encrypted mail support that you can follow here: https://github.com/elementary/mail/issues/82

We already ship full disk encryption with LUKS which is the standard for Linux-based operating systems.

We also already ship support for OpenVPN and WireGuard out of the box

We're not able to ship 3rd party applications by default, but there is a Bitwarden web app wrapper available from Flathub. There is a built-in system password manager, but perhaps we need a graphical way to manage those passwords. It would be worth opening a specific feature request about a password manager, probably here: https://github.com/elementary/switchboard-plug-security-privacy/issues

There's an open feature request for "shred" here: https://github.com/elementary/files/issues/1048

There's an open feature requests here for backup: https://github.com/elementary/os/issues/266

I definitely recommend searching for or creating other individual feature requests since this discussion post won't be able to be easily searched or tracked

**Reply by wasantosfi** [reply]
Hi @danirabbit thank you for your reply.

Regarding "lightweight" kernel, I was trying to say a smaller one, without old drivers (eg.: very old processors), maybe without dial modems drivers.
Some examples of lightweight distros are AntiX, Zorin OS Lite, Linux Lite and LXLE.

About "bloatware" for a Minimal version, we could leave as options during install procedure apps like Mail, Calendar, Code and PDF Viewer. Once some users can use online resources to mails, calendar, pdfs, and great part of devs uses VS Code (good to integrate to Snyk, SonarLint, Contrast Security and other security resources).
 
I wasn't aware about Brave's history, thank you for the info.

About Mail app and Thunderbird, the opened feature request for encryption is the good answer, great! But look like anyone is working on it for a while.

LUKS do the job for disk encryption (I use it here in my 7.1 Horus), just trying to put an alternative on table in order to improve cross platform for external drives.

I'll need to check OpenVPN configuration here, because it only works with "official" client (OpenVPN3).

Regarding File Shredder, looks like no one was looking to it for some years. 

The same about backup (I use Timeshift, and it worked pretty fine when I tried to update my GPU driver (Nvidia MX230) and the system refused to bootup).

Thank you for your recommendation, I'll try to collaborate properly.

**Comment by araujo791** [comment]
It's a good idea to have the basic installation option, without applications like camera, email, tasks...

**Comment by teamcons** [comment]
The idea of spins is interesting

A privacy one feels like redundant effort. There are a lot of distributions focused on this aspect, who will always do it better, and all the softwares listed are there for who wants to install. I can bet someone else has a different idea of what should be in that you or me and would deem some of the preinstalled listed ones "bloat" too.

The mini one is neat ! But ultimately sound a bit... Off to me, considering the OS exist to ship the DE and its suite, offering the OS without half of its homegrown suite, spending efforts to ship it despite the efforts to do a full suite and experience, seem a bit pointless/self defeating ?
Like theres also "ikea" OSes who do it better. I do not see the added value here

**Comment by doobes** [comment]
Agree completely to the lighter install.

Frankly I find virtually none of the prepackaged apps useful. 

I delete all/most of them and install the apps I want. Doing a quick check on this machine, none of the installed apps remain.

With the limited resources available my suggestion would be to minimize development time on the apps and spend it on the OS. 

Cheers
